client/src/styles/ApplicantProfile.css

```diff
@@ -1,7 +1,7 @@
-body {
+/* body {
```

Comment:
If you do not need this code just remove it.

Reply:
this code i didn't add it is one of the group members so i just hide it so i can see the actual colours coming from the components